templates.yaml

```diff
@@ -209,6 +212,10 @@
     Type: "Frontend"
     Tools: "Next.js, TypeScript, HTML, CSS"
     License: "Apache-2.0"
+  inputs:
+    agentName:
+      type: string
+      description: "Optional - dispatch only agents with this name"
```

Comment:
is it possible to include a help link? https://docs.livekit.io/agents/worker/agent-dispatch

i'd also try to align the copy more with how it's referenced in the docs. "Set agent name to enable explicit dispatch" or something like that